tests/dataloader_tests.cpp

```diff
@@ -50,19 +50,23 @@ BOOST_AUTO_TEST_CASE(CSVDataLoaderTest)
   // Check for training dataset using tuples.
   BOOST_REQUIRE_EQUAL(std::get<0>(irisDataloader.TrainSet()).n_cols, 75);
   BOOST_REQUIRE_EQUAL(std::get<0>(irisDataloader.TrainSet()).n_rows, 4);
+
+  Utils::RemoveFile("./../data/iris.csv");
 }
 
 /**
  * Simple test for MNIST Dataloader.
  */
 BOOST_AUTO_TEST_CASE(MNISTDataLoaderTest)
 {
+  /**
   DataLoader<> dataloader("mnist", true, 0.80);
   // Check for correct dimensions.
   BOOST_REQUIRE_EQUAL(dataloader.TrainFeatures().n_cols, 784);
   BOOST_REQUIRE_EQUAL(dataloader.TestFeatures().n_cols, 784);
   BOOST_REQUIRE_EQUAL(dataloader.ValidFeatures().n_cols, 784);
   BOOST_REQUIRE_EQUAL(dataloader.TrainFeatures().n_rows, 33600);
+  */
```

Comment:
Should we keep this commented block?

Reply:
Once the mnist_train / test.csv in in mlpack.org/datasets we can uncomment this.